### PR TITLE
Add test for newborn stats

### PR DESCRIPTION
**Pull request by MaxHalford**

### CODE REVIEW

File: river/stats/mode.py [hunk 1]
```diff
@@ -62,7 +62,9 @@ def update(self, x):
         return self
 
     def get(self):
-        return max(self.counts, key=self.counts.get)
+        if self.counts:
+            return max(self.counts, key=self.counts.get)
+        return None
```
Comment (smastelini) [inline river/stats/mode.py:67]: What about using:

`return max(self.counts, key=self.counts.get, default=None)`?

File: river/stats/quantile.py [hunk 2]
```diff
@@ -161,7 +161,7 @@ def get(self):
             length = len(self.heights)
             return self.heights[int(min(max(length - 1, 0), length * self.q))]
 
-        return 0
+        return None
```
Comment (smastelini) [inline river/stats/quantile.py:164]: Nice catch!

File: river/stats/test_.py [hunk 3]
```diff
@@ -48,6 +51,12 @@ def test_pickling(stat):
...
 
+@pytest.mark.parametrize("stat", load_stats(), ids=lambda stat: stat.__class__.__name__)
+def test_repr_with_no_updates(stat):
+    assert isinstance(repr(stat), str)
+    assert isinstance(str(stat), str)
+
+
 @pytest.mark.parametrize(
     "stat, func",
     [
```
Comment (smastelini) [inline river/stats/test_.py:62]: Nice!!

**Review by smastelini — CHANGES_REQUESTED** [review]
Great work @MaxHalford! The new test is a must for pipeline viz! I left one small comment that might result in changes.

The PR is in good shape!

File: river/stats/mode.py (hunk 1 above)
Comment (MaxHalford) [reply river/stats/mode.py:67]: Oh nice!

**Pull request merged**